fixes issues with current Issue Templates that were created in haste (#14)
sandialabs/Gulliver: .github/ISSUE_TEMPLATE/bug_report.md

---

# Describe the bug

A clear and concise description of what the bug is. 
If available include the exception that is thrown, including the type and the included message and stack trace.

# Expected behavior

A clear and concise description of what you expected to happen.

# Example of how to Reproduce

Provide the steps to reproduce the behavior by including a code example either inline or via a [Gist](https://gist.github.com/) link.
If a code example is not possible, or too complex for the context of the issue describe in detail

# Code Information

 - Runtime Information [e.g. netcore2.0, netstandard2.1]
 - SDK / Runtime Version [e.g. Core 2.2 SDK 2.2.402]
 - Language Version [e.g. C# 7.3, F# 4.6]
 - Operating System [e.g. Linux, Windows 10, Mac]

# Proposed solutions

> Do you have a solution, what would you recommend to fix it?

Reviewer: This is awkwardly worded / punctuated. For whatever reason an alternative is not immediately springing to mind, but lets do better.

> Do you have a solution, what would you recommend to fix it?

Reviewer: ```suggestion
Do you have a solution that may fix this issue?
```

# Can you help?

> Let us know if you are interested in helping, and in what capacity. eg: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc

Reviewer: ```suggestion
Let us know if you are interested in helping, and in what capacity. e.g.: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc.
```

> 

Reviewer: ```suggestion
```
